docs/DeveloperGuide.adoc

```diff
@@ -182,7 +182,8 @@ Step 2. The user executes `delete 5` command to delete the 5th recipe in the rec
 
 image::UndoRedoState1.svg[]
 
-Step 3. The user executes `add n/David ...` to add a new recipe. The `add` command also calls `Model#commitRecipeBook()`, causing another modified Recipe book state to be saved into the `RecipeBookStateList`.
+Step 3. The user executes `add n/Chicken Rice ...` to add a new recipe. The `add` command also calls
```

Comment:
```suggestion
Step 3. The user executes `new n/Chicken Rice ...` to add a new recipe. The `new` command also calls
```

Reply:
I thought it was `new` as well but I realized this is a suggested functionality

Reply:
On reading it again, I think you are right. Will update the Dev Guide to reflect this.

Thanks!